tests/unit/assessment/test_pipelines.py

```diff
@@ -21,14 +27,76 @@ def test_pipeline_assessment_with_config(mocker):
         )
     ]
 
-    ws = Mock()
+    ws = create_autospec(WorkspaceClient)
     config_dict = {
         "spark.hadoop.fs.azure.account.auth.type.abcde.dfs.core.windows.net": "SAS",
         "spark.hadoop.fs.azure.sas.token.provider.type.abcde.dfs."
         "core.windows.net": "org.apache.hadoop.fs.azurebfs.sas.FixedSASTokenProvider",
         "spark.hadoop.fs.azure.sas.fixed.token.abcde.dfs.core.windows.net": "{{secrets/abcde_access/sasFixedToken}}",
     }
+    pipeline_cluster = [
+        PipelineCluster(
+            apply_policy_default_values=None,
+            autoscale=None,
+            aws_attributes=None,
+            azure_attributes=None,
+            cluster_log_conf=None,
+            custom_tags={'cluster_type': 'default'},
+            driver_instance_pool_id=None,
+            driver_node_type_id=None,
+            gcp_attributes=None,
+            init_scripts=[
+                InitScriptInfo(
+                    dbfs=DbfsStorageInfo(destination="dbfs:/users/test@test.com/init_scripts/test.sh"),
+                    s3=None,
+                    volumes=None,
+                    workspace=None,
+                )
+            ],
+            instance_pool_id=None,
+            label='default',
+            node_type_id='Standard_F4s',
+            num_workers=1,
+            policy_id="test_id",
+            spark_conf={"spark.databricks.delta.preview.enabled": "true"},
+            spark_env_vars=None,
+            ssh_public_keys=None,
+        )
+    ]
     ws.pipelines.get().spec.configuration = config_dict
+    ws.pipelines.get().spec.clusters = pipeline_cluster
+    ws.cluster_policies.get().definition = (
```

Comment:
refactor to start using `workspace_client_mock` - see https://github.com/databrickslabs/ucx/blame/main/tests/unit/assessment/test_clusters.py#L22-L58

Reply:
replaced it with workspace_client_mock